### PR TITLE
Update the release notes to include all the releases that happened since the last update

### PR DESCRIPTION
**Pull request by pavel-constantinov**

### CODE REVIEW

File: content/en/docs/apidocs-mxsdk/apidocs/extensibility-api/release-notes.md [hunk 1]
```diff
@@ -17,33 +17,35 @@ These release notes cover changes to the Extensibility API.
...
-* We renamed the `MicroflowService UpdateActionAfterRename` method to `RenameVariable`.
-* We enable error dialogs for extensions when using the `--enable-extension-development` feature flag.
+* No user facing changes, still please keep the extension package version synced with StudioPro.
 
-## Version 10.6.0
+## Version 10.12.1
+
+* No user facing changes, still please keep the extension package version synced with StudioPro.
+
+## Version 10.12.0
```
Comment (MarkvanMents) [inline content/en/docs/apidocs-mxsdk/apidocs/extensibility-api/release-notes.md:50]: I don't really understand why we are saying that 10.12.0 is the first beta when we released a beta on version 10.6.0 and published release notes for versions between 10.6 and 10.12.

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Hi @pavel-constantinov 
I'm sorry I haven't worked with this before so these may be stupid questions.

- It is not clear to me where all these different versions are available. I've looked in the [GitHub repo](https://github.com/mendix/ExtensionAPI-Samples) and there are no releases or packages set up there.
- Our documentation also doesn't point anywhere else to obtain the API, or to see all the calls available.
- I'm also not sure what "please keep the extension package version synced with Studio Pro" actually means? Does it mean that you have to use version 10.15 with Studio Pro 10.15? I can't see any documentation anywhere on how to update the version of the Extension APIs when you upgrade your Mendix app.

**Comment by pavel-constantinov** [comment]
> Hi @pavel-constantinov I'm sorry I haven't worked with this before so these may be stupid questions.
> 
> * It is not clear to me where all these different versions are available. I've looked in the [GitHub repo](https://github.com/mendix/ExtensionAPI-Samples) and there are no releases or packages set up there.
> * Our documentation also doesn't point anywhere else to obtain the API, or to see all the calls available.
> * I'm also not sure what "please keep the extension package version synced with Studio Pro" actually means? Does it mean that you have to use version 10.15 with Studio Pro 10.15? I can't see any documentation anywhere on how to update the version of the Extension APIs when you upgrade your Mendix app.

Hi @MarkvanMents, no worries, I'll do my best to bring you up to speed.

- Firstly, as it is a NuGet package it's available at nuget.org, the readme of that package is actually pointing to this release notes page.
- Secondly, it certainly does, not necessarily directly but our docs mention that the package is hosted on nuget.org.  
image

  The wording could be improved and a link to nuget.org wouldn't hurt, but this is unrelated to this story.
- Finally, our package is meant to be updated through the NuGet tool in the user's IDE of choice.

File: content/en/docs/apidocs-mxsdk/apidocs/extensibility-api/release-notes.md (hunk 1 above)
Comment (pavel-constantinov) [reply content/en/docs/apidocs-mxsdk/apidocs/extensibility-api/release-notes.md:50]: I'm pretty sure 10.12 was the public beta release, not 10.6 as that was the closed beta for partners. So there's no point in mentioning the stuff up until 10.12, as it is practically impossible to get or use those versions of the package, they're simply not there.

Comment (MarkvanMents) [reply content/en/docs/apidocs-mxsdk/apidocs/extensibility-api/release-notes.md:50]: OK, I'll resolve this conversation.

**Review by MarkvanMents — APPROVED** [review]
I've changed the wording a bit. Please check it still makes sense.
Approving - let me know if it OK and I'll merge it.

**Comment by MarkvanMents** [comment]
> > Hi @pavel-constantinov I'm sorry I haven't worked with this before so these may be stupid questions.
> > 
> > * It is not clear to me where all these different versions are available. I've looked in the [GitHub repo](https://github.com/mendix/ExtensionAPI-Samples) and there are no releases or packages set up there.
> > * Our documentation also doesn't point anywhere else to obtain the API, or to see all the calls available.
> > * I'm also not sure what "please keep the extension package version synced with Studio Pro" actually means? Does it mean that you have to use version 10.15 with Studio Pro 10.15? I can't see any documentation anywhere on how to update the version of the Extension APIs when you upgrade your Mendix app.
> 
> Hi @MarkvanMents, no worries, I'll do my best to bring you up to speed.
> 
>     * Firstly, as it is a NuGet package it's available at nuget.org, the readme of that package is actually pointing to this release notes page.
> 
>     * Secondly, it certainly does, not necessarily directly but our docs mention that the package is hosted on nuget.org.
> 
> 
> image
> 
> The wording could be improved and a link to nuget.org wouldn't hurt, but this is unrelated to this story.
> 
>     * Finally, our package is meant to be updated through the NuGet tool in the user's IDE of choice.

Thanks for the update.
I'll add a task to look at whether we can make the use of NuGet a bit clearer. Now you have explained it I see that it is mentioned, but as someone who has never used NuGet it didn't stand out to me.

I've done a small proofread. Hope it still looks good. Let me know if I can merge. https://github.com/mendix/docs/pull/8373/commits/d803f09258db1da943050800822293553226f2af

**Comment by MarkvanMents** [comment]
Approved over Slack.

**Pull request merged**